openedx/features/user_leads/helpers.py

```diff
@@ -15,6 +26,15 @@ def get_utm_params(request):
 
 
 def save_user_utm(request):
+    """
+
+    Args:
+        request: HttpRequest object
+
+    Returns:
+        None
+
```

Comment:
Same a) description b) blank line

Reply:
Yes, there are a few extra blank lines, but a blank line between last line of args and "Returns" is recommended.

Edx Recommendation:

```
def func(arg1, arg2):
    """
    Summary line.

    Extended description of function.

    Arguments:
        arg1 (int): Description of arg1
        arg2 (str): Description of arg2

    Returns:
        bool: Description of return value
    """
```

@zeshanarifios can we use "Arguments" in the docstring instead of "Args" because the former is recommended by edx.

Reply:
Also can we list the arguments as shown above, according to edx recommendations?